### DISCUSSION TITLE
Extra permission required?

### DISCUSSION THREAD

**Discussion by mboon75** [shibayan/appservice-acmebot#345] [Q&A] [answered]
Hi, we are using the azure function already for some time and yesterday i discovered a couple of errors. It looks that it has to do with permissions. Is something changed there?

The client 'x' with object id 'y' has permission to perform action 'Microsoft.Web/sites/write' on scope '/subscriptions/z/resourceGroups/ResourceGroup/providers/Microsoft.Web/sites/appservice'; however, it does not have permission to perform action 'join/action' on the linked scope(s) '/subscriptions/z/resourceGroups/OtherResourceGroup/providers/Microsoft.Network/virtualNetworks/NameOfNetwork/subnets/Portals' or the linked scope(s) are invalid.

**Comment by shibayan** [comment, marked as answer]
Acmebot only requires permissions related to App Service and App Service Plan, but when using VNET Integration, additional permissions may be required due to ARM API limitations.

I have not tested it in that scenario, so you will need to follow the error message and add the necessary permissions.
